### PR TITLE
HW7 is done and ready to code review

### PR DESCRIPTION
**Pull request by lexahnure**

### CODE REVIEW

File: date.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  const thisDate = `${date.toDateString()}, ${date.getHours()}:${date.getMinutes()}`;
+
+  console.log(thisDate);
+}
+getDate();
+
+const path = require('path');
+const getFileName = () => {
+  const fileName = path.basename(__filename);
+  return console.log(fileName);
```
Comment (iliatcymbal) [inline date.js]: why need return statement here?

File: date.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const getDate = () => {
+  const date = new Date();
+  const thisDate = `${date.toDateString()}, ${date.getHours()}:${date.getMinutes()}`;
+
+  console.log(thisDate);
+}
+getDate();
```
Comment (iliatcymbal) [inline date.js:8]: it is good practice to make empty line after block with constants and variables

File: getRN.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+const getRandomNumber = (min = 1, max = 5) => {
+  const number = Math.floor(Math.random() * (max - min)) + min;
+  return console.log(number);
+}
+
+module.exports = (from, to) => () => getRandomNumber(from, to);
```
Comment (iliatcymbal) [inline getRN.js:6]: any reason to use here function that returns function?

File: read.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+const fs = require('fs');
+
+const read = (name) => {
+  return new Promise ((res, rej) => {
```
Comment (iliatcymbal) [inline read.js]: short style for arrow function (no need to use _return_)

**Review by iliatcymbal — APPROVED** [review]
please, see my notes. I would appreciate if you fix them

File: getRN.js (hunk 3 above)
Comment (lexahnure) [reply getRN.js:6]: because without returning function code doesn't work. and you wrote the same code in class

**Pull request merged**